JS/app.js

```diff
@@ -0,0 +1,49 @@
+let searchButton = document.getElementById("searchButton").addEventListener('click', Search);
+let loadingSpot = document.getElementById("loading")
+let loadingIndicator = `<div class="d-flex justify-content-center"><div class="spinner-grow text-info" role="status">
+<span class="sr-only">Loading...</span>
+</div></div>`;
+
+function Search(e) {
+    e.preventDefault();
+    let searchTerm = document.getElementById("searchTerm").value;
+    getInstruments(`https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${searchTerm}&limit=10&exchange=NASDAQ`);
+}
+
+async function getInstruments(url) {
+    showResults();
+    loadingSpot.innerHTML = loadingIndicator;
+    try {
+        const response = await fetch(url);
+        const result = await response.json();
+        loadingSpot.innerHTML = "";
+        result.forEach(item => {
```

Comment:
what happens if there is no results? Show some nice error to the user

Reply:
Error handling added to this functionality inside SearchForm class.